strongbox-storage/strongbox-storage-indexing/src/main/java/org/carlspring/strongbox/services/impl/ConfigurationManagementServiceImpl.java

```diff
@@ -87,18 +91,20 @@ public void setProxyConfiguration(String storageId,
                                       ProxyConfiguration proxyConfiguration)
             throws IOException, JAXBException
     {
+        Configuration configuration = configurationManager.getConfiguration();
         if (storageId != null && repositoryId != null)
         {
-            configurationManager.getConfiguration()
+            configuration
```

Comment:
I meant, change the line from:
    
    configuration
                                 .getStorage(storageId)

To:

    configuration.getStorage(storageId)

and so on...

(The same applies to the similar comments below).